internal/provider/virtualmachine/schema_virtualmachine.go

```diff
@@ -23,6 +23,11 @@ func Schema() map[string]*schema.Schema {
 			Optional: true,
 			Computed: true,
 		},
+		constants.FieldVirtualMachineReservedMemory: {
+			Type:     schema.TypeString,
+			Optional: true,
+			Computed: true,
+		},
```

Comment:
```suggestion
			Optional: true,
		},
```

Reply:
Appreciated for the test and consideration. I have updated according to the suggestion